src/oas.md

```diff
@@ -1087,6 +1087,10 @@ schema:
     type: integer
     format: int64
 style: simple
+examples:
+  number:
+    dataValue: 12345678
+    serializedValue: "token: 12345678"
```

Comment:
Can we add a second example, demonstrating the use of an array coming from deserializing the simple style?

```suggestion
examples:
  one number:
    dataValue: 12345678
    serializedValue: "token: 12345678"
  two numbers:
    dataValue:
      - 12345678
      - 99999999
    serializedValue: "token: 12345678,99999999"
```

Reply:
@karenetheridge upon further thought, I'm going to split this PR up to separate the new fields from all of the extra example updates.  I'll take this request into consideration then- I think that might be better than either allowing endless new example requests here or trying to keep them out of this process entirely.